webapi/src/test/scala/org/knora/webapi/e2e/v1/ResourcesV1R2RSpec.scala

```diff
@@ -1250,45 +1177,181 @@ class ResourcesV1R2RSpec extends R2RSpec {
                 }
 
                 assert(utf8str == "another simple text")
+            }
+        }
 
+        "create resources from an XML import" in {
+            val xmlImport =
+                s"""<?xml version="1.0" encoding="UTF-8"?>
+                   |<knoraXmlImport:resources xmlns="http://api.knora.org/ontology/biblio/xml-import/v1#"
+                   |    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   |    xsi:schemaLocation="http://api.knora.org/ontology/biblio/xml-import/v1# biblio.xsd"
+                   |    xmlns:biblio="http://api.knora.org/ontology/biblio/xml-import/v1#"
```

Comment:
Both `http://ecommerce.example.org/schema` and `http://ecommerce.example.org/schema#` are valid XML namespaces, [and they are considered different XML namespaces](https://stackoverflow.com/a/431315).

In RDF, a namespace and a local part are concatenated, so you have to decide whether the namespace should end with `#` or `/`. Thus in RDF you have [either a 'hash namespace' or a 'slash namespace'](https://www.w3.org/2001/sw/BestPractices/VM/http-examples/2006-01-18/#naming).

But in XML, the namespace and the local part are not concatenated. The namespace and the local part are two separate things, which together uniquely identify an element. There is no concept in XML of concatenating them together to make a single URI, as there is in RDF. See [this question and the answers](http://lists.xml.org/archives/xml-dev/201207/msg00308.html).

However, we would like our API to be based on dereferenceable ontology entities, at least in version 2. So I think we should use RDF namespace conventions in the XML import. Then if someone tries to dereference `http://api.knora.org/ontology/biblio/xml-import/v2#JournalArticle`, they should get some information about that class. I don't want to implement this in v1, but for consistency I'd like to use the same naming convention there, too.